### ISSUE TITLE
SA1504: Additional setter lines are deleted unintentionally

### ISSUE THREAD

**Issue by daflotsch** [DotNetAnalyzers/StyleCopAnalyzers#1448]
If I have a something like:

```csharp
public Uri CurrentContentUri
{
    get { return _currentContentUri; }
    private set
    {
        _currentContentUri = value;
        OnPropertyChanged();
    }
}
```

and want to follow the suggestion for SA1504, it will delete the OnPropertyChanged() line. It seems like it deletes all additional setter lines on this rule

![image](https://cloud.githubusercontent.com/assets/14220636/9793236/b46bb6b0-57e4-11e5-86b2-7a4ebc44fe24.png)

**Comment by pdelvo** [comment]
Wow thats a bad one. Thanks for reporting. Im taking a look at it.

**Comment by daflotsch** [comment]
Thanks a lot, appreciate it!
